src/core/radio/trel_interface.cpp

```diff
@@ -352,6 +352,24 @@ void Interface::RemovePeerEntry(Peer &aEntry)
     mPeerTable.PopBack();
 }
 
+void Interface::SendPacket(otInstance       *aInstance,
+                           const uint8_t    *aUdpPayload,
+                           uint16_t          aUdpPayloadLen,
+                           const otSockAddr *aDestSockAddr)
+{
+    otPlatTrelSend(aInstance, aUdpPayload, aUdpPayloadLen, aDestSockAddr); // mCounters
```

Comment:
Remove `// mCounters` ?

Reply:
Sorry, the previous commit was a draft, removed this comment. I still want to keep mCounters variable because otGetCounters needs to return a const pointer which is kinda s standard thing.